charts/kubeai/templates/service.yaml

```diff
@@ -11,5 +11,9 @@ spec:
       targetPort: http
       protocol: TCP
       name: http
+    - port: 8080
```

Comment:
```suggestion
    - port: {{ .Values.service.metricsPort }}
```

and in values.yaml set default to "8080"

Reply:
The metrics port is configurable in KubeAI so we should also expose this in the service object.

Reply:
actually ignore this, we don't have a way to override this anyway in the helm chart. Resolving this comment.